receiver/sapnetweaverreceiver/webservice.go

```diff
@@ -114,13 +121,37 @@ func (s *netweaverWebService) ABAPGetSystemWPTable() (*models.ABAPGetSystemWPTab
 	return response, nil
 }
 
-func (s *netweaverWebService) GetRequestLogFile() (*models.GetRequestLogFileResponse, error) {
-	request := &models.GetRequestLogFile{}
-	response := &models.GetRequestLogFileResponse{}
+func (s *netweaverWebService) OSExecute(command string) (*models.OSExecuteResponse, error) {
+	request := &models.OSExecute{
+		Command: command,
+	}
+	response := &models.OSExecuteResponse{}
 	err := s.client.Call("''", request, response)
 	if err != nil {
 		return nil, err
 	}
 
 	return response, nil
 }
+
+func (s *netweaverWebService) FindFile(args ...string) ([]string, error) {
```

Comment:
if we're using OSExecute to execute the commands, it implies that this could be remote collection, in which case we should use OSExecute to find the files too.

Reply:
Basically we should either use OSExecute everywhere or nowhere.

Reply:
I can use OSExecute here.   I'll test to see if I can get the certificates without the OS Execute. Then we could remove OSExecute all together.

Reply:
Updated to completely remove OSExecute.

Reply:
This means that the receiver needs to be run on the host it's targeting now, right? Is that acceptable for users? If so, we should add a note about that to the README.